packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -183,6 +183,9 @@ export class GasEstimator {
     if (this.type == NetworkType.London) {
       this.latestMaxFeePerGasGwei = (gasInfo as LondonGasData).maxFeePerGas;
       this.latestMaxPriorityFeePerGasGwei = (gasInfo as LondonGasData).maxPriorityFeePerGas;
+      if (this.latestMaxPriorityFeePerGasGwei > this.latestMaxFeePerGasGwei)
+        this.latestMaxFeePerGasGwei = this.latestMaxPriorityFeePerGasGwei / 1.5;
```

Comment:
Maybe I'm missing something, but is this supposed to change which one is larger? In my reading of this code, latestMaxFeePerGasGwei is checked to see if it's smaller and then the math just ensures that it remains smaller than latestMaxPriorityFeePerGasGwei?

Can you explain what's going on here?

Also, if we're doing math on these, we may want to round them off at a certain number of decimals to ensure we don't end up getting errors when it can't be converted to wei down below. We saw errors related to this in the across v2 relayer. You can use `Number(value.toFixed(decimals))` to do this.

Reply:
We just nede to make sure maxFeePerGas is higher than maxPriorityFee per gas, so I actually have the direction wrong I think this should be * not /